setup.cfg

```diff
@@ -46,3 +46,9 @@ all-files = 1
 source-dir = docs/
 build-dir = docs/_build
 warning-is-error = 1
+
+[options]
+packages = find_namespace:
+
+[options.packages.find]
+include = ["openff.*"]
```

Comment:
Presumably one or both of these sections is what fixed the `UserWarning: Module openff was already imported from None,` warning? I assumed it was simply something weird on my local machine given that I do unholy things with local installs and don't look back ...